routes/__tests__/petitions.test.js

```diff
@@ -1,9 +1,49 @@
-const app = require("../../app");
-const request = require("supertest");
+const app = require('../../app');
+const request = require('supertest');
+const { runQuery, getConnection } = require('../../lib/sql');
 
-describe("/petitions", () => {
-  test("it is available", async () => {
-    const response = await request(app).get("/petitions");
-    expect(response.status).toBe(200);
-  });
+describe('/petitions', () => {
+    beforeEach(async () => {
+        app.db = await getConnection(
+            `${__dirname}/../../db/node_api_exercise_tests.sqlite`
+        );
+    });
+    test('it is available', async () => {
+        const response = await request(app).get('/petitions');
+        expect(response.status).toBe(200);
+    });
+
+    test('it returns all the petitions', async () => {
```

Comment:
In the Rails version there’s no `returns all petitions test` written for them, and I like that it lets the candidate decide how they want to test (interact with the provided test database, vs importing a library).

What are your thoughts on leaving in the import and `beforeEach`, but removing it returns all the petitions? Having the import should be a hint to candidates that there is a test DB for them to interact with.

Reply:
Removing the test also works for me.
Note that the code needs the db injection that happens in the `beforeEach`. 
Could be moved (hidden) into jest async config.

Reply:
Did that ^